.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "image": "ghcr.io/insightsengineering/rstudio:latest",
+  "name": "RStudio Dev Container",
+  
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  // "features": {},
+  
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  // "forwardPorts": [],
+  
+  // Use 'postCreateCommand' to run commands after the container is created.
+  // "postCreateCommand": "uname -a",
+  
+  // Configure tool-specific properties.
+  "customizations": {
+    "vscode": {
+      "extensions": [
```

Comment:
what's your view on air formatter - it might require additional config file so it's fine for this to go separately

Reply:
I meant this: https://marketplace.visualstudio.com/items?itemName=Posit.air-vscode